tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -271,233 +271,233 @@
-    <string name="pref_tor_enforce_distinct_subnets" translatable="false">EnforceDistinctSubnets</string>
-    <string name="pref_tor_enforce_distinct_subnets_summ">If enabled, Tor will not put two servers whose IP addresses are "too close" on the same circuit. Currently, two addresses are "too close" if they lie in the same /16 range.</string>
-    <string name="pref_tor_enable_SOCKS_proxy">Enable SOCKS proxy</string>
-    <string name="pref_tor_SOCKS_port" translatable="false">SOCKSPort</string>
-    <string name="pref_tor_SOCKS_port_summ">Open this port to listen for connections from SOCKS-speaking applications.</string>
-    <string name="pref_tor_enable_HTTPTunnel_proxy">Enable HTTPTunnel</string>
-    <string name="pref_tor_HTTPTunnel_port" translatable="false">HTTPTunnelPort</string>
+    <string name="pref_tor_entry_nodes_alt_summ">Turn off \"Tor bridges\" if you want to select entry nodes yourself.</string>
+    <string name="pref_tor_strict_nodes" translatable="false">Strict Nodes</string>
+    <string name="pref_tor_strict_nodes_summ">Adheres to \"Exclude Nodes\" for all generated Tor circuits, even if it breaks functionality.</string>
+    <string name="pref_tor_fascist_firewall" translatable="false">Fascist Firewall</string>
+    <string name="pref_tor_fascist_firewall_summ">Makes Tor create only outgoing connections to ORs running on ports that your firewall allows (defaults to 80 and 443).</string>
+    <string name="pref_tor_new_circuit_period" translatable="false">New Circuit Period</string>
+    <string name="pref_tor_new_circuit_period_summ">Considers builing a new circuit every set amount of seconds.</string>
+    <string name="pref_tor_max_circuit_dirtiness" translatable="false">Max Circuit Dirtiness</string>
+    <string name="pref_tor_max_circuit_dirtiness_summ">Reusable circuits first used at most the set amount of seconds ago, but never attach a new stream to a too old circuit.</string>
+    <string name="pref_tor_enforce_distinct_subnets" translatable="false">Enforce Distinct Subnets</string>
+    <string name="pref_tor_enforce_distinct_subnets_summ">Makes Tor not put two servers whose IP addresses are "too close" (in the same /16 range) on the same circuit.</string>
+    <string name="pref_tor_enable_SOCKS_proxy">SOCKS Proxy</string>
+    <string name="pref_tor_SOCKS_port" translatable="false">SOCKS Port</string>
+    <string name="pref_tor_SOCKS_port_summ">Open this port to listen for connections from SOCKS-speaking apps.</string>
+    <string name="pref_tor_enable_HTTPTunnel_proxy">HTTP Tunnel</string>
+    <string name="pref_tor_HTTPTunnel_port" translatable="false">HTTP Tunnel Port</string>
     <string name="pref_tor_HTTPTunnel_port_summ">Open this port to listen for proxy connections using the "HTTP CONNECT" protocol instead of SOCKS.</string>
```

Comment:
```suggestion
    <string name="pref_tor_HTTPTunnel_port_summ">Open this port to listen for proxy connections using the \"HTTP CONNECT\" protocol instead of SOCKS.</string>
```